docs/fieldnorm/verification_validation.md

```diff
@@ -1,7 +1,70 @@
-# Verification and validation
+# Software Verification & Software Validation
```

Comment:
One thing I'm still missing is what falls under software in terms of "software" validation.
For example, how do you handle the validation of a reference pool, reference genome or a database update. Does this fall under software validation? software verification? or does this belong to the ISO15189 way of performing validations or verification?

Also, what about updates? major/minor/hotfix what falls under what? I'm still a bit hazy on what should be validated and what would suffice under verification.

Maybe we could add a list of examples?

Reply:
up for discussion:
we need to do a release when a new databases or reference set is used. We do this in a hotfix way, i.e. changing the 3rd number.. 
so then ,we do need people to check whether changes are properly included, and the validation is often a single sentence...